src/components/UI/CardsStyled/Cards.styled.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+export const CardContainer = styled.li`
+	display: flex;
+	flex-direction: column;
+	align-self: center;
+	padding: 30px;
+	margin-top: 70px;
+	border-radius: 20px;
+	width: 33rem;
```

Comment:
same here, let the parent decide how much space this card gets.

Reply:
Fixed